integrations/integrations.go

```diff
@@ -163,11 +166,21 @@ func startMonomerNode(
 		return fmt.Errorf("create CometBFT listener: %v", err)
 	}
 
-	blockdb, err := dbm.NewDB("blockstore", dbm.BackendType(svrCtx.Config.DBBackend), svrCtx.Config.RootDir)
+	var blockPebbleDB *pebble.DB
+	if backendType := dbm.BackendType(svrCtx.Config.DBBackend); backendType == dbm.MemDBBackend {
+		blockPebbleDB, err = pebble.Open("", &pebble.Options{
+			FS: vfs.NewMem(),
+		})
+	} else {
+		if backendType != dbm.PebbleDBBackend {
+			svrCtx.Logger.Info("Overriding provided db backend for the blockstore", "provided", backendType, "using", dbm.PebbleDBBackend)
+		}
+		blockPebbleDB, err = pebble.Open("blockstore", nil)
+	}
```

Comment:
Should we still use `svrCtx.Config.RootDir` as the pebble directory name whenever the DB is opened?

Reply:
Ahhh good point. I think you're right. Will investigate.